app/src/main/java/com/devshish/internship/presentation/service/player/MediaSessionCallback.kt

```diff
@@ -29,6 +39,42 @@ class MediaSessionCallback(
         playWhenReady = true
     }
 
+    private fun NotificationCompat.Builder.createNotification(): NotificationCompat.Builder {
+        return apply {
```

Comment:
You do not need here apply, because you've already created  `NotificationCompat.Builder.` extension function and you already have this scope